### ISSUE TITLE
Add node command to CLI

### ISSUE THREAD

**Issue by ericnordelo** [OpenZeppelin/nile-rs#12]

**Comment by ftupas** [comment]
Hello there, may I take this one?

**Comment by ericnordelo** [comment]
Hi @ftupas. Glad you are interested in contributing. You can take it if you want, but keep in mind that devnet is still implemented in python, so it will be difficult to integrate the command for the rust CLI in a clean way (no dependency version management). I would wait for a rust implementation of devnet for providing a proxy directly through nile.

**Comment by ftupas** [comment]
> Hi @ftupas. Glad you are interested in contributing. You can take it if you want, but keep in mind that devnet is still implemented in python, so it will be difficult to integrate the command for the rust CLI in a clean way (no dependency version management). I would wait for a rust implementation of devnet for providing a proxy directly through nile.

Yeah that makes sense, I think it's best to wait for the rust implementation of devnet! I'll probably take the other issue for now, if that's alright 😄
